validate/validate.go

```diff
@@ -145,12 +163,15 @@ func byteCheckRtmr(field string, size int, given, required [][]byte) error {
 		return fmt.Errorf("RTMR field size(%d) is not equal to expected size(4)", len(required))
 	}
 	for i := range required {
+		if len(required[i]) == 0 {
+			continue
+		}
 		if len(required[i]) != size {
 			return fmt.Errorf("RTMR[%d] should be 48 bytes, found %d", i, len(required[i]))
 		}
 		if !bytes.Equal(required[i], given[i]) {
-			return fmt.Errorf("quote field %s is %s. Expect %s",
-				field, hex.EncodeToString(given[i]), hex.EncodeToString(required[i]))
+			return fmt.Errorf("quote field %s[%d] is %s. Expect %s",
```

Comment:
"quote field" is less specific than then RMTR[%d] you use in the error above. Should the RTMR/48 text be replaced with %s[%d] <- field, size? Or can you just drop the field argument since the function is clearly for the RTMR flag?

Reply:
Done. Dropped field argument